packages/react-core/src/components/Banner/Banner.tsx

```diff
@@ -0,0 +1,33 @@
+import * as React from 'react';
+import styles from '@patternfly/react-styles/css/components/Banner/banner';
+import { css } from '@patternfly/react-styles';
+
+export interface BannerProps extends React.HTMLProps<HTMLButtonElement> {
+  /** Content rendered inside the banner */
+  children?: React.ReactNode;
+  /** Additional classes added to the banner */
+  className?: string;
+  /** Variant styles for the banner */
+  variant?: 'default' | 'info' | 'danger' | 'success' | 'warning';
+  /** If set to true, the banner sticks to the top of its container */
+  isSticky?: boolean;
+}
+
+export const Banner: React.FunctionComponent<BannerProps> = ({
+  children,
+  className,
+  variant = 'default',
+  isSticky = false
+}: BannerProps) => (
+  <div
+    className={css(
+      styles.banner,
+      styles.modifiers[variant as 'success' | 'danger' | 'warning' | 'info'],
```

Comment:
do we need to check for `variant !== default` here?

Reply:
If `variant === default` no class gets applied, so I don't think we need to check for it.